addon/components/au-button.js

```diff
@@ -3,14 +3,14 @@ import Component from '@glimmer/component';
 export default class AuButton extends Component {
   get widthClass() {
     if (this.args.width == "block")
-      return "au-c-input--block";
+      return "au-c-button--block";
     else
       return "";
   }
 
   get skinClass() {
-    if (this.args.error == "ghost")
-      return "au-c-input--ghost";
+    if (this.args.skin == "ghost")
+      return "au-c-button--ghost";
```

Comment:
Does this project adhere to braceless control structures? That's very odd, this is probably the first time I've seen it in this format.

Reply:
Expanding on what I meants:
```javascript
if (this.args.skin == "ghost") {
    return "au-c-input--ghost";
} else {
  ...
}
```

What I used to do a long time ago is to in-line all braceless structures like so:
```javascript
if (this.args.skin == "ghost") return "au-c-input--ghost";
```

I'm just mentioning this out of curiosity, as it's out of the scope of this specific issue

Reply:
Thanks for the suggestion. I am not sure what the best approach is here. I've seen a bunch of different ones, but I am not the expert here. Love to see what the best practice is :-)!